hotxlfp/grammarparser/lexer.py

```diff
@@ -53,7 +47,7 @@ def t_STRING(t):
 
 
 def t_FUNCTION(t):
-    r'([A-Za-z]{1,}[A-Za-z_0-9\.]+(?=[(]))|([A-Za-z\.]+(?=[(]))'
+    r'([A-Za-z]{1,}[A-Za-z_0-9\.]+(?=[(]))|([A-Za-z\.]+(?=\s+[(]))'
```

Comment:
will this cause problems if i write "A (B)" where A and B are variables? did we support implied multiplication before?

Reply:
Discussed live